macOS/LocalPackages/DataBrokerProtectionShared/Sources/DataBrokerProtectionSharedTestsUtils/Mocks.swift

```diff
@@ -1419,14 +1380,56 @@ public final class MockDataBrokerOperationErrorDelegate: DataBrokerOperationErro
     }
 }
 
+public final class MockOperationEventsHandler: EventMapping<OperationEvent> {
+
+    public var profileSavedFired = false
+    public var firstScanCompletedFired = false
+    public var firstScanCompletedAndMatchesFoundFired = false
+    public var firstProfileRemovedFired = false
+    public var allProfilesRemovedFired = false
+
+    public init() {
+        super.init { _, _, _, _ in
+        }
+
+        // A workaround to be able to reference self in the eventMapper
```

Comment:
I'm not very happy with this, so we should discuss a better alternative.

This instance being a mock is relatively low stakes, but I think it makes more difference for the pixel handlers so they don't have to use the static pixelKit methods

Reply:
We discussed this sync and there isn't a better way to do it...